hm-service-account-api-rest-v1.yml

```diff
@@ -636,3 +670,43 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Error'
+  FleetFrobidenErrors:
+    type: object
+    title: Errors
+    properties:
+      errors:
+        description: List of errors
+        type: array
+        items:
+          $ref: '#/definitions/Error' 
+        example:
+          - detail: This service account is not associated with a fleet.
```

Comment:
Is the service account or application not associated?

Reply:
the clients are creating a service account token and use it the token. That's why I put service account. 

Do you prefer to use `This application  is not associated with a fleet.` ? or another text

Reply:
Yep, in case the application ID is in the parameters.

Reply:
only `ServiceAccountAccessToken` is in the parameters and some data related to the vehicles.

Reply:
Ah yes the service account key is already created under an app. Can keep the original text.